packager/transformer.js

```diff
@@ -17,15 +17,24 @@ const json5 = require('json5');
 const path = require('path');
 const ReactPackager = require('./react-packager');
 
-const babelRC =
-  json5.parse(
-    fs.readFileSync(
-      path.resolve(__dirname, 'react-packager', '.babelrc')));
 
-function transform(src, filename, options) {
-  options = options || {};
+const projectBabelRCPath = path.resolve(__dirname, '..', '..', '..', '.babelrc');
```

Comment:
Thinking about this some more, I think the babelrc under the path specified by `--projectRoots` is what we want to use. That way if you invoke the packager and point it at another directory, we'd correctly use that directory's .babelrc.